.travis.yml

```diff
@@ -322,6 +321,23 @@ matrix:
       - cat /tmp/out.txt
       - sudo cat /var/log/squid3/cache.log
       - sudo cat /var/log/squid3/access.log
+  - rvm: 2.4.4
```

Comment:
Can we get a brief comment here at to what this test is. The description in the PR is pretty solid